### PR TITLE
Left menu buttons Styling

### LINKED ISSUE #1485: Left menu item button design improvement
The left hand menu buttons now have designs in Figma with rest, hover and active states:

Image

[Figma designs](https://www.figma.com/design/RGRSmIC4KoVlIosQB5YmQY/%F0%9F%93%B1%F0%9F%A7%B1-App-components?m=auto&node-id=971-5497&t=Tdl2zDMnlDxe3Ans-1)

### PR DESCRIPTION
**Pull request by Al12rs**
- fix #1485

### CODE REVIEW

File: src/NexusMods.App.UI/LeftMenu/Items/IconViewModel.cs [hunk 1]
```diff
@@ -15,5 +15,5 @@ public class IconViewModel : AViewModel<IIconViewModel>, IIconViewModel
 
     [Reactive] public string[] Badges { get; set; } = [];
 
-    [Reactive] public ReactiveCommand<NavigationInformation, Unit> NavigateCommand { get; set; } = ReactiveCommand.Create<NavigationInformation>(_ => { }, Observable.Return(false));
+    [Reactive] public ReactiveCommand<NavigationInformation, Unit> NavigateCommand { get; set; } = ReactiveCommand.Create<NavigationInformation>(_ => { }, Observable.Return(true));
```
Comment (erri120) [inline src/NexusMods.App.UI/LeftMenu/Items/IconViewModel.cs:18]: Why did you change this?

Comment (erri120) [reply src/NexusMods.App.UI/LeftMenu/Items/IconViewModel.cs:18]: To give some context, I originally put the default value to `false` because anyone that wants to add navigation has to supply their own command anyways.

Comment (Al12rs) [reply src/NexusMods.App.UI/LeftMenu/Items/IconViewModel.cs:18]: Ah it was mostly for preview purposes, so I could test the hover states. I can revert it if you want but, if it doesn't hurt I would leave it like that for the preview.

**Review by erri120 — APPROVED** [review]
